pom.xml

```diff
@@ -1,9 +1,11 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>disparse</groupId>
+    <groupId>com.github.BoscoJared</groupId>
```

Comment:
Not 100% crazy with this change as reverse domain is really nice when you are an entity i.e. Facebook has com.facebook.* but for a small open-source project having my username / full name in every package seems undesirable.

Reply:
alright, fair enough